sources/TestView.cpp

```diff
@@ -199,9 +203,14 @@ TestView::MessageReceived(BMessage* msg)
 				if (fType != TEST_LOCATION)
 					flavors |= B_FILE_NODE;
 
+				BString prefix = B_TRANSLATE("Choose");
+				prefix << ' '
+					<< B_TRANSLATE(fType == TEST_LOCATION ? "folder" : "file")
+					<< ' ' << B_TRANSLATE("for") << ' ';
+
```

Comment:
Here, too, we need to create the string first and ReplaceFirst() the various variables.